ballerina/LICENSE

```diff
@@ -15,193 +15,193 @@ below is and where the actual text of the license can be found.
 Name                                                                                                Type           License             
 _________________________________________________________________________________________________________________________________________________
 ballerina-command-1.3.14.jar                                                                        jar            apache2
-xml-2201.5.0.jar                                                                                    jar            apache2
 commons-text-1.10.0.jar                                                                             bundle         apache2
 compiler-0.8.9.jar                                                                                  jar            apache2
-openapi-core-1.5.0.jar                                                                              jar            apache2
-semver-checker-cli-2201.5.0.jar                                                                     jar            apache2
-transaction-2201.5.0.jar                                                                            jar            apache2
+ballerina-cli-2201.5.1-20230616-211200-8f5066c0.jar                                                 jar            apache2
```

Comment:
@udda1996 Shouldn't we ideally have the jar's stable versions in the license file? Here we seem to have timestamped versions for most of the jars